client/Contexts/ConfigFeatureFlag/ConfigFeatureFlag.spec.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import {
+  ConfigFeatureFlagProvider,
+  ConfigFeatureFlagConsumer,
+} from './Context';
+
+describe('ConfigFeatureFlag context', () => {
+  describe('initial state tests', () => {
+    it('returns the expected configuration values', () => {
+      const { getByText } = render(
+        <ConfigFeatureFlagProvider>
+          <ConfigFeatureFlagConsumer>
+            {({ config }) => {
+              return <p>{JSON.stringify(config)}</p>;
+            }}
+          </ConfigFeatureFlagConsumer>
+        </ConfigFeatureFlagProvider>
+      );
+      expect(getByText('{"example":{"value":true}}')).toBeInTheDocument();
```

Comment:
This will be changing in the next PR, where with the introduction of the apollo pieces, seeding real 'mocked' response data was a requirement for the provider to be tested properly. I also have made some changes to the tests to use more helper code, akin to your `setUp` example in that PR

Reply:
https://github.com/matthew-chirgwin/strimzi-ui/tree/13-integrate/client/Contexts/ConfigFeatureFlag